.github/workflows/Test.yml

```diff
@@ -36,6 +36,14 @@ jobs:
   jvm:
     runs-on: ${{ matrix.os }}
 
+    services:
+      mysql:
```

Comment:
Do we really want to go back to pulling down an image instead of using the existing mysql in the image?

Reply:
We could, but we should:
a) use 5.7 (or update that separately rather than implying a version based on the runner), otherwise we cause a mysql update breaking builds: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-software
b) use an action to set it up: https://github.com/mirromutth/mysql-action#the-default-mysql